curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/653281af14be5f2055310f8e.md

```diff
@@ -7,16 +7,28 @@ dashedName: step-8
 
 # --description--
 
-Create a `userData` object that will contain the songs, the current song playing, and the time of the current song.
+Your music player should keep track of the songs, the current song playing, and the time of the current song. To do this, you will need to create an object to store this information.
 
-Declare an empty `userData` object using the `let` keyword.
+Start by using the `let` keyword to declare a new variable called `userData`. Assign an empty object to your `userData` variable.
```

Comment:
Thanks for your contribution. Make this change and I'll gladly approve.

```suggestion
Start by using the `let` keyword to declare a new variable called `userData` and assign it an empty object.
```

Reply:
resolved